src/Controller/EthereumController.php

```diff
@@ -92,9 +105,11 @@ public static function getServerOptionsArray($filter_enabled = FALSE){
    * Returns Ethereum Servers.
    *
    * @param bool $filter_enabled
-   *   (optional) Restrict to enabled servers. Defaults to FALSE.
+   *    Restrict to enabled servers. Defaults to FALSE (optional).
```

Comment:
Drupal coding standards say that (optional) needs to come first, like it was before :)

Reply:
ok